src/main/java/com/amihaiemil/zold/Wallet.java

```diff
@@ -71,11 +72,17 @@ void pay(
     /**
      * Finds all payments that match this query and returns.
      * @return Array of found transactions.
-     * @param id Wallet id
-     * @param details Regex of payment details
+     * @param params Json object contains params to search for.
+     *   Possible values:
```

Comment:
@ammaratef45 I think it would be better to use javadoc HTML formatting here:
```
 * possible values:
 * <ul>
 *   <li>id</li>
 *  ...
 * </ul>
```